tracing-subscriber/src/registry/fmt_layer.rs

```diff
@@ -0,0 +1,197 @@
+use crate::{
+    fmt::{format, FormatEvent, FormatFields, MakeWriter},
+    layer::{Context, Layer},
+    registry::{Extensions, LookupMetadata, LookupSpan, Registry, SpanData, SpanRef},
+};
+use ansi_term::{Color, Style};
+use humantime;
```

Comment:
i like that this crate now depends on `chrono` _and_ `humantime` :P

can we eventually replace this with the existing timestamp formatting code?

Reply:
lmao, yes. i removed that!